demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -0,0 +1,64 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import sys
+import cv2
+import numpy as np
+from argparse import ArgumentParser, SUPPRESS
+
+from openvino.inference_engine import IECore
+
+from inpainting import ImageInpainting
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model.",
+                      required=True, type=str)
+    args.add_argument("-i", "--input", type=str, default='', help="path to image.")
+    args.add_argument("-d", "--device",
+                      help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
+                           "acceptable. The demo will look for a suitable plugin for device specified. "
+                           "Default value is CPU", default="CPU", type=str)
+    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask.", default=8)
```

Comment:
```suggestion
    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask.", default=8, type=int)
```
The same goes for other int keys below.